### ISSUE TITLE
Why the training loss of CT-segmentation is negative?

### ISSUE THREAD

**Issue by ayf-9797** [MIC-DKFZ/nnUNet#2018]
I have experienced all negative values for training loss and validation loss when training with both 3d-full and 2d modes. However, all preprocessing operations were done all according to the instructions. What is the cause of this situation and will it affect the prediction results of the network?
such as:
![image](https://github.com/MIC-DKFZ/nnUNet/assets/76467367/6051b302-71fb-4674-bc6b-fe16ab770544)

**Comment by ayf-9797** [comment]
Thanks to the author for maintaining this section, I have found the answer I was expecting.
